kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -121,6 +120,8 @@ def digest(self) -> str:
 
     @cached_property
     def llvm_dylib(self) -> Path | None:
+        from .config import Kernel
```

Comment:
I think you only need these local imports to make the unit test work, the code movements is unnecessary.

```suggestion
        from .kompile import Kernel
```